src/main/kotlin/uk/gov/justice/hmpps/architecture/annotations/Tags.kt

```diff
@@ -16,7 +16,10 @@ enum class Tags : addTo {
   },
   PLANNED,
   DEPRECATED,
-  SOFTWARE_AS_A_SERVICE;
+  SOFTWARE_AS_A_SERVICE,
+  AUTH_API,
+  PROBATION_API,
+  PRISONS_API;
```

Comment:
_Idea_ Can we use only `API` and use `ProblemArea` to reflect the domain?

Reply:
Runtime only, but we can also infer `*_API` from the existence of `APIDocs`

Reply:
💭 Idea from our chat:

Use `API` for APIs and `AREA_PRISON`, `AREA_PROBATION`?

Reply:
I've partitioned the tags as above and added area and data views